composer.json

```diff
@@ -1,32 +1,75 @@
 {
-  "name": "cwp/cwp-installer",
-  "description": "CWP Project Template",
-  "license": "BSD-3-Clause",
-  "require": {
-    "cwp/cwp-recipe-basic": "1.8.x-dev",
-    "cwp/cwp-recipe-blog": "1.8.x-dev",
-    "cwp/starter-theme": "1.1.x-dev"
-  },
-  "require-dev": {
-    "cwp/cwp-recipe-basic-dev": "^1.1@stable"
-  },
-  "suggest": {
-    "silverstripe/textextraction": "Required for document text searching within CWP"
-  },
-  "config": {
-    "process-timeout": 900
-  },
-  "extra": {
-    "branch-alias": {
-      "dev-master": "1.8.x-dev"
-    }
-  },
-  "repositories": [
-    {
-      "type": "composer",
-      "url": "https://packages.cwp.govt.nz/"
-    }
-  ],
-  "prefer-stable": true,
-  "minimum-stability": "dev"
+    "name": "cwp/cwp-installer",
+    "type": "silverstripe-recipe",
+    "description": "CWP Project Template",
+    "license": "BSD-3-Clause",
+    "require": {
+        "php": ">=5.6.0",
+        "silverstripe/recipe-plugin": "^1",
+        "silverstripe/recipe-core": "1.1.x-dev",
+        "silverstripe/assets": "1.1.x-dev",
+        "silverstripe/config": "1.0.x-dev",
+        "silverstripe/framework": "4.1.x-dev",
```

Comment:
If we're using the silverstripe/recipe-core we don't need to list these here as well

Reply:
That's me misinterpreting your [earlier comment](https://github.com/silverstripe/cwp-installer/pull/5#discussion_r164947291). Including just the recipe seems much more logical, I'll strip it back again.